app/assets/stylesheets/application.scss

```diff
@@ -1,7 +1,13 @@
 .navbar-brand {
-    position: absolute;
-    width: 100%;
-    left: 0;
-    text-align: center;
-    margin:0 auto;
+  position: absolute;
+  width: 100%;
+  left: 0;
+  text-align: center;
+  margin:0 auto;
+}
+
+.top-movie-title {
+  width: 100%;
+  text-align: center;
+  margin: 2% auto 3% auto;
 }
```

Comment:
Can you do this with a bootstrap class instead? (I don't know the answer to this)